style.css

```diff
@@ -1 +1,285 @@
-/*  Add your styling here */
+:root {
+
+}
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    font-family: 'Montserrat', sans-serif;
+}
+
+nav {
+    display: none;
+}
+
+/* HEADER   */
+
+.header-top-section {
+    background-color: rgb(247, 247, 247);
+    display: flex;
+    justify-content: space-between;
+    height: 150px;
+    padding: 20px 20px 10px 20px;
+    gap: 100px;
+}
+
+.header-align {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    align-content: space-between;
+    justify-content: space-between;
+}
+
+.logo {
+    width: 80px;
+    height: 80px;
+}
+
+h1{ 
+    font-size: 0.75em;
+    font-weight: 300;
+    text-align: right;
+    /* padding: 0 30px 30px 0; */
```

Comment:
Try and not include commented out code in PRs

Reply:
Keep missing them, thanks will make sure to double check my code and omit them from here on out, thanks :)